### PR TITLE
Longer testbed facts gathering timeout - 2h

### PR DESCRIPTION
**Pull request by sodar**
As far as I know (and what docs show), ansible cannot do infinite loops.

### CODE REVIEW

File: ansible/roles/vmware-vm/tasks/post_deploy.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     folder: "{{ datacenter_name }}/vm/{{ testenv_folder }}"
     name: "{{ inventory_hostname_short }}"
   register: newvm
-  retries: 15
+  retries: 600
```
Comment (krdln) [inline ansible/roles/vmware-vm/tasks/post_deploy.yml]: Let's do one hour for now (perhaps 1.5h) - we think that if after one hour it doesn't have an IP, the problem is somewhere else.

(Also, some jobs as `deploy-demo-env` use this ansible role without any uppper timeout)

**Comment by sodar** [comment]
@krdln I have forgotten how to multiply xD

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
Started `deploy-builder` job: http://10.84.12.26:8080/job/WinContrail-infra/job/deploy-builder/134/

**Comment by sodar** [comment]
recheck no bug

**Pull request merged**